tests/framework-tests/preferences/js/PanelTestUtils.js

```diff
@@ -42,4 +45,10 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         jqUnit.assertEquals(message, expectedState, checkbox.is(":checked"));
     };
 
+    fluid.tests.panels.utils.getTemplate = function (that) {
```

Comment:
Not sure what you mean by this? It's used by every component except linksControls